static/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+ <meta charset="utf-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+ <title>Drupalcamp Cebu 2018</title>
+
+ <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+ <link rel="stylesheet" href="/assets/style.css">
+ <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
```

Comment:
This is not the fonts we are using. Please use Google fonts to configure the new site.